tests/cli/test_pipeline_command.py

```diff
@@ -207,9 +207,19 @@ def test_pipeline_command_drop_partial_loads(repo_dir: str, project_files: FileS
     os.environ["EXCEPTION_PROB"] = "1.0"
     os.environ["FAIL_IN_INIT"] = "False"
     os.environ["TIMEOUT"] = "1.0"
+
     venv = Venv.restore_current()
     with pytest.raises(CalledProcessError) as cpe:
         print(venv.run_script("chess_pipeline.py"))
+
+    # move job into running folder manually
```

Comment:
see comment about this below

Reply:
above actually

Reply:
the comment right above. `def is_package_partially_loaded` returns False if all jobs in a normalized package are in the started folder. So if I have a test that needs `is_package_partially_loaded` to return true, I have to move one job into the running or completed folder. Before my changes, the job that needed retrying remained in the started_jobs folder so this was not needed. There are two places where I test this. My question above is wether we should change is_package_partially_loaded to return true also if there was at least one failed retry.